src/styles/blocks/body.scss

```diff
@@ -0,0 +1,9 @@
+body {
```

Comment:
it's better to name this file common.scss. Since you can add some styles fro html tag or something, so it'd more flexible